src/video-player.js

```diff
@@ -510,6 +510,11 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       }
     });
 
+    this.videojs.on('canplaythrough', () => {
+      // clear retry timeout
```

Comment:
In my opinion, the comment can be removed.. the function named 'clearTimeout' is clean enough